modules/misc/qt.nix

```diff
@@ -221,7 +215,6 @@ in
                 "lxqt"
                 "qtct"
                 "kde"
-                "kde6"
```

Comment:
Should probably have a graceful deprecation warning instead of rug-pulling users, no?

Reply:
Yeah, you're right. I thought of that, but wasn't sure how to go about it. Could you please give an example of how this is usually handled?

Reply:
I roughed out a deprecation function that applies to the `platformTheme` section. It warns when it has `kde6` and converts it to `kde`

Reply:
Thanks for the help!